sklearn/model_selection/_split.py

```diff
@@ -367,6 +367,10 @@ class KFold(_BaseKFold):
         If None, the random number generator is the RandomState instance used
         by `np.random`. Used when ``shuffle`` == True.
 
+    .. deprecated:: 0.20
```

Comment:
this should be "versionchanged" I think? Maybe? The parameter is not deprecated, only the default value changed.